astroquery/skyview/core.py

```diff
@@ -241,95 +156,89 @@ def get_image_list(self, position, survey, *, coordinates=None,
         Examples
         --------
         >>> SkyView().get_image_list(position='Eta Carinae',
-        ...                          survey=['Fermi 5', 'HRI', 'DSS'])
-        [u'http://skyview.gsfc.nasa.gov/tempspace/fits/skv6183161285798_1.fits',
-         u'http://skyview.gsfc.nasa.gov/tempspace/fits/skv6183161285798_2.fits',
-         u'http://skyview.gsfc.nasa.gov/tempspace/fits/skv6183161285798_3.fits']
+        ...                          survey=['HRI', 'DSS'])
+        ['https://skyview.gsfc.nasa.gov/cgi-bin/images?position=161.26474075372%2C-59.6844587777
+          &survey=dss&pixels=300%2C300&sampler=LI&size=1.0%2C1.0&projection=Tan
+          &coordinates=J2000.0&requestID=skv1696445832839&return=FITS']
         """
 
         self._validate_surveys(survey)
 
-        if radius is not None:
-            size_deg = str(radius.to(u.deg).value * 2)
-        elif width and height:
-            size_deg = "{0},{1}".format(width.to(u.deg).value,
-                                        height.to(u.deg).value)
-        elif width and height:
-            raise ValueError("Must specify width and height if you "
-                             "specify either.")
-        else:
-            size_deg = None
-
-        input = {
-            'Position': parse_coordinates(position),
-            'survey': survey,
-            'Deedger': deedger,
-            'lut': lut,
-            'projection': projection,
-            'gridlabels': '1' if gridlabels else '0',
-            'coordinates': coordinates,
-            'scaling': scaling,
-            'grid': grid,
-            'resolver': resolver,
-            'Sampler': sampler,
-            'imscale': size_deg,
-            'size': size_deg,
-            'pixels': pixels}
-        response = self._submit_form(input, cache=cache)
-        urls = self._parse_response(response)
-        return urls
-
-    def _parse_response(self, response):
-        bs = BeautifulSoup(response.content, "html.parser")
         urls = []
-        for a in bs.find_all('a'):
-            if a.text == 'FITS':
-                href = a.get('href')
-                urls.append(urlparse.urljoin(response.url, href))
+        for current_survey in survey:
+            registry_results = registry.search(ivoid=self._survey_dict[current_survey]['ivoid'])
```

Comment:
I don't think there is a need for another call to the registry here. During the first call in `_survey_dict`, you can also save `access_url`, and then use it here to create the ` sia_service` by something like: ` sia_service  = pyvo.sia.SIAService(self._survey_dict[current_survey]['access_url'])`